docs/topics/js/js-overview.md

```diff
@@ -3,66 +3,63 @@
 Kotlin/JS provides the ability to transpile your Kotlin code, the Kotlin standard library, and any compatible dependencies
 to JavaScript. The current implementation of Kotlin/JS targets [ES5](https://www.ecma-international.org/ecma-262/5.1/).
 
-The recommended way to use Kotlin/JS is via the `kotlin.js` and `kotlin.multiplatform` Gradle plugins. They provide a
-central and convenient way to set up and control Kotlin projects targeting JavaScript. This includes essential functionality
+The recommended way to use Kotlin/JS is via the `kotlin.js` and `kotlin.multiplatform` Gradle plugins. They let you easily set up and control Kotlin projects targeting JavaScript in one place. This includes essential functionality
 such as controlling the bundling of your application, adding JavaScript dependencies directly from npm, and more. To get
 an overview of the available options, check out the [Kotlin/JS project setup](js-project-setup.md) documentation.
 
 ## Use cases for Kotlin/JS
 
-There are numerous ways that Kotlin/JS can be used. To provide you some inspiration, here's a non-exhaustive list of
-scenarios in which you can use Kotlin/JS.
+There are numerous ways to use Kotlin/JS. Here is a non-exhaustive list of
```

Comment:
Just a note: there are too much words highlighted with bold.
I guess we could eliminate these highlighitings

Reply:
Of course, we shouldn't do it in this PR.